content/en/docs/examples/vm-bookinfo/index.md

```diff
@@ -1,14 +1,16 @@
 ---
-title: Bookinfo with Mesh Expansion
-description: Illustrates how to expand the Bookinfo application's mesh with a raw VM service.
+title: Bookinfo on Virtual Machines
+description: Run the Bookinfo application on virtual machines within your mesh.
```

Comment:
This title and description are totally wrong. This example is not running Bookinfo on virtual machines. It is running Bookinfo on Kubernetes and adding one dependent service (MySQL), running on a VM. The difference between this and the similar egress approach (https://deploy-preview-5215--preliminary-istio.netlify.com/blog/2018/egress-tcp/) is that in this example the VM service is being added to the mesh, instead of accessed as an external service. It is "expanding the mesh" with a VM service.

Reply:
I agree with @frankbu that a different title is probably better to make clear that this is just adding a service running in a VM to a microservice-architected app running on Kubernetes. However, the left navigation doesn't like long titles, which limit the amount of nuance I can introduce. Would the following alternatives be better?
```suggestion
title: Bookinfo with Virtual Machines
description: Run the Bookinfo application with a MySQL service running in a virtual machine within your mesh.
```

Reply:
I'd prefer to fix the title to make its clear, e.g. "Adding a VM to a mesh"

"Mesh Expansion" is a wording I've **never** heard from users to describe how to enmesh VMs, despite our docs saying it for a while. So I'm suggesting we stop using that specific word. ("Mesh Expansion" doesn't even say "VM" anywhere in it)

Having an example that had bidirectional traffic from the VM would be even better, but we don't have to hold this change for having that.

Reply:
Or maybe "Bookinfo with adding VM-based services"

Reply:
I'm not so fond of the term "Mesh Expansion" either, not sure who coined it. That said, I think the important point is that this is about "adding a non-k8s service to a k8s mesh". If we can come up with a general term that we add to the glossary (like we did with "Multicluster", "Multi mesh", etc.), then the title of this one could just be "Bookinfo <general term>".

Since the added service could be running on bare metal or a VM (it's all just a matter of network connectivity), the general term should not just be VM-based. "multi-runtime mesh", "heterogeneous runtime mesh",  I don't know.

Reply:
I think we shouldn't invent a new term. VMs are a very common use case, and these docs are specifically about VMs. I think we should correctly describe how this example works with a VM, **using the term VM**. We could put as a side-note that these instructions also work for bare metal, if we want to.

I hear over and over from customers that they don't believe Istio supports VMs. (I never hear "Istio doesn't support bare metal") I'm specifically advocating for using "VM" so that it's clear that we do support VMs.